src/pyedb/dotnet/edb_core/cell/hierarchy/component.py

```diff
@@ -56,9 +57,9 @@ class EDBComponent(object):
 
     """
 
-    def __init__(self, pedb, cmp):
-        self._pedb = pedb
-        self.edbcomponent = cmp
+    def __init__(self, pedb, edb_object):
```

Comment:
Just to know, what's the difference here between `pedb` and `edb_object` ? The first is the `Edb` instance while the second is any object from the API ?

Reply:
pedb is pyedb.edb.Edb

edb_object is EDB API object. In this case, it is   Ansys.Ansoft.Edb.Cell.Hierarchy.Component.